web/index.html

```diff
@@ -39,67 +39,71 @@ <h3 class="section-title">About Me</h3>
         <h3 class="section-title">Let's connect</h3>
       </c>
       <c span="row"></c>
-      <c span="row">
+      <c span="row" span-tablet="1+4" span-desktop="1+8">
         <div class="image-wrapper">
           <img class="avatar" src="image/avatar-raluca.jpg" alt="">
         </div>
       </c>
-      <c span="row"></c>
-      <c span="1">
-        <a class="disc-link__onahau-dark">
-          <i class="my-cv-udemy"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__granny-apple">
-          <i class="my-cv-codewars"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__oasis">
-          <i class="my-cv-codetap"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__onahau-light">
-          <i class="my-cv-facebook"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__blue-chalk">
-          <i class="my-cv-github"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__pastel-pink">
-          <i class="my-cv-skype"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__azalea">
-          <i class="my-cv-linkedin"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__moon-raker">
-          <i class="my-cv-stackoverflow"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__white-ice">
-          <i class="my-cv-instagram"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__janna">
-          <i class="my-cv-twitter"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__double-pearl-lusta">
-          <i class="my-cv-youtube"></i>
-        </a>
-      </c>
+      <c span="row" span-tablet="4" span-desktop="8">
```

Comment:
I am not up to date with the project. but is a bit wired to see `c` instead of `div` ... I assume `c` stands for `column`

Reply:
Yes, `c`  stands for `column` and I think this was Marian's way of doing just the starter version of my-cv-v2 just for the purpose of better explaining the grid layout to the us, the starters 😄; hope I'm not wrong